### ISSUE TITLE
Number of own pulled in dependencies

### ISSUE THREAD

**Issue by therealprof** [est31/cargo-udeps#13]
The goal of the project is very laudable but I have the feeling it's falling a bit short by having one of the longest depedencies sets of all projects I've worked with (including really big ones) with 172 crates...

**Comment by est31** [comment]
I agree that the number of dependencies is too high. Currently, cargo-udeps needs the cargo library crate which is cause of most of the dependencies. This is so that we have similar familiar CLI args to cargo and roughly the same behaviour. Either we fix it before #4, e.g. by using cargo build plans, or we switch to alternatives to save-analysis and remove it then.

**Comment by est31** [comment]
cc https://github.com/oli-obk/cargo_metadata/issues/46
